coalib/bearlib/aspects/Spelling.py

```diff
@@ -63,54 +63,3 @@ class docs:
     specific_word = Taste[str](
         'Represents the regex of the specific word to check.',
         ('c[o|O][a|A][l|L][a|A]', ), default='')
-
-
-@Spelling.subaspect
-class aspectsYEAH:
```

Comment:
Oh no...
I think few of my new test cases are dependent on this and `coalaCorrect` :(